wasm/Cargo.toml

```diff
@@ -22,33 +22,49 @@ crate-type = [ "cdylib", "rlib" ]
 doctest = false
 
 [dependencies.snarkvm-circuit-network]
-version = "=0.16.12"
+version = "0.16.19"
+git = "https://github.com/AleoNet/snarkVM.git"
+rev = "3ebe60c499285a140ec2171fe8dd06662c325531"
```

Comment:
Current Rev is: 140ff26 (I know this changed from under us since this process started)

Reply:
Absolutely, I will update it before this gets merged.